ios/Classes/OfflineRegionData.swift

```diff
@@ -9,13 +9,13 @@ import Foundation
 import Mapbox
 
 class OfflineRegionData {
```

Comment:
might be a good idea to rename this to OfflineRegion? So that the naming between dart and swift is consistent

Reply:
Sure 👍